### PR TITLE
Enable GitHub Actions, and disable Travis

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Test
+on: push
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```
Comment (vietnguyen-td) [inline .github/workflows/build.yml]: could you change to the latest one? https://github.com/actions/checkout

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
```
Comment (vietnguyen-td) [inline .github/workflows/build.yml]: ditto

**Review by vietnguyen-td — COMMENTED** [review]
left concern about version

**Comment by dmikurube** [comment]
Thanks for your good catch! Fixed.

**Review by vietnguyen-td — APPROVED** [review]
LGTM

**Pull request merged**